test/fixtures/basic/src/scripts.js.uglify-js.concat

```diff
@@ -1,11 +0,0 @@
----
-files:
-- script.js
-# Put additional JavaScript in "files".
-mangle: true
----
-
-// This is scripts.js.uglify-js.concat
-var name = 'World'
-var message = 'Hello ,' + name + '!'
-console.log(message)
```

Comment:
I think the basic test should be fixed now. You're right that it was broken before. Bringing in the latest from `master` should fix it.